server/enterprise.go

```diff
@@ -173,16 +174,6 @@ func (s *Server) createEnterpriseTestsErrorStatus(ctx context.Context, pr *model
 		"Failed running enterprise tests. @mattermost/core-build-engineers have been notified. Error:  \n```"+err.Error()+"```")
 }
 
-func (s *Server) succeedOutDatedJenkinsStatuses(ctx context.Context, pr *model.PullRequest) {
```

Comment:
Why did we remove this? (Just to have a bit of context)

Reply:
The transition phase is over, so it should be good to remove.